src/python/bot/fuzzers/blackbox/engine.py

```diff
@@ -81,18 +88,12 @@ def prepare(self, corpus_dir, target_path, build_dir):  # pylint: disable=unused
     Returns:
       A FuzzOptions object.
     """
-    app_path = environment.get_value('APP_PATH')
-    app_args = testcase_manager.get_command_line_for_application(
-        get_arguments_only=True).strip()
-
-    arguments = _get_arguments(app_path, app_args)
-    arguments.append(f'--input_dir={corpus_dir}')
-
+    executable_path = environment.get_value('FUZZER_EXECUTABLE_PATH')
```

Comment:
Rather than use an environment variable to hack around the interface, can we just pass the full path to the fuzzer as target_path instead? 

Right now target_path is fuzzer_name, but there's really no reason we need to care about what the name of the fuzzer is here. It's only used to get the full path to the fuzzer anyway, so let's just compute the full path beforehand and pass it here.